src/Validators/NameValidator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Portal\Validators;
+
+class NameValidator
+{
+    public static function validateExistsAndLength(string $validateName, int $characterLength)
+    {
+        if (empty($validateName) == false && strlen($validateName) <= $characterLength) {
+            return $validateName;
+        } else {
+            throw new \Exception('An input string does not exist or is too long');
+        }
+    }
```

Comment:
maybe make a StringValidator abstract with this method in it and extend it for NameValidator, PriceValidator and SKUValidator?